src/renderer/utils/__tests__/jsonPath.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { parseJsonPath } from "../jsonPath";
+
+describe("parseJsonPath", () => {
+  test("should convert \. to use indexed notation", () => {
+    const res = parseJsonPath(".metadata.labels.kubesphere\\.io/alias-name");
+
+    expect(res).toBe(".metadata['labels']['kubesphere.io/alias-name']");
+
+  });
+
+  test("strips '\' away from the result", () => {
+    const res = parseJsonPath(".metadata.labels['serving\\.knative\\.dev/configuration']");
+
+    expect(res).toBe(".metadata.labels['serving.knative.dev/configuration']");
+  });
+
```

Comment:
there should be a test for the following case: `".metadata.labels\.serving['some.other.item']"`

Reply:
Added test case for this